docs/adoption.md

```diff
@@ -188,6 +188,24 @@ Finally, in GitHub actions secrets, set `AZUREAPPSERVICE_PUBLISHPROFILE_PRODUCTI
 
 ### Deploy housing-repairs-online-api
 
+#### Github Action logs Masking Secrets
```

Comment:
This section is relevant for all APIs and would be better placed under the [Deploying the API's](https://github.com/Newarkandsherwood/housing-repairs-online/pull/14/files?diff=split&w=1#diff-fbf6f356c391aa4359bb6635f637548d890a6390f7e28338957bd16d368b8d2aR120) heading.

Reply:
This section is currently relevant only to housing-repairs-online-api since this is the only API that requires specific extra secrets to be entered. It goes on to list all of the secrets that must be added in a table.

Reply:
This section is specifically for Terraform generated resources that also need masking.
The generic secret masking documentation has also been [added](https://github.com/Newarkandsherwood/housing-repairs-online/pull/14/files#diff-303fac764df0be46899474eecc8eb1be10d8893b8f7b0f4cbccbb16ae9097033R15).